### PR TITLE
Use js 'standard' formatter on pre-commit

### LINKED ISSUE #157: Adopt JS Standard Style
I normally don't care about this kind of stuff but I got a few questions about this during the Code + Learn. We currently use some kind of 4 space indent that confused some people.

It might make sense to adopt standard, https://github.com/feross/standard . It's a sensible style and has tools that relieve us of having to enforce the style manually.

### PR DESCRIPTION
**Pull request by fhemberger**
Closes #157 and #159.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "serve": "node build.js serve",
     "lint": "eslint scripts build.js",
```
Comment (phillipj) [inline package.json]: Remove the old lint script and explicit eslint dep below?

Comment (fhemberger) [inline package.json]: Aye!

**Comment by phillipj** [comment]
LGTM

**Comment by phillipj** [comment]
`.eslintrc` could be deleted aswell

**Comment by fhemberger** [comment]
Removed `.eslintrc`.

File: scripts/helpers/changeloglink.js [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
+  if (!version) { return '' }
 
-    if (!version) { return ''; }
-
-    return semver.gte(version, '1.0.0') ?
-        `https://github.com/nodejs/io.js/blob/${version}/CHANGELOG.md` :
-        `https://github.com/joyent/node/blob/${version}/ChangeLog`;
-};
+  return semver.gte(version, '1.0.0')
+    ? `https://github.com/nodejs/io.js/blob/${version}/CHANGELOG.md`
```
Comment (evanlucas) [inline scripts/helpers/changeloglink.js:9]: shouldn't this be `https://github.com/nodejs/node/blob/${version}/CHANGELOG.md`?

File: scripts/helpers/changeloglink.js [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
+    : `https://github.com/joyent/node/blob/${version}/ChangeLog`
```
Comment (evanlucas) [inline scripts/helpers/changeloglink.js]: Should this link to `nodejs/node-v0.x-archive`?

Comment (fhemberger) [inline scripts/helpers/changeloglink.js]: Yes, the script has been used the last time before the v4.0.0 release. Fixing this …

**Comment by fhemberger** [comment]
@nodejs/website Should we merge this?

**Comment by mikeal** [comment]
+1

**Comment by zeke** [comment]
:clap:

**Pull request merged**